# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/rustc/mips64.patch
# Copyright (C) 2022 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- rustc-1.72.0-src/compiler/rustc_target/src/spec/targets/mips64_unknown_linux_gnuabi64.rs.orig	2023-08-29 19:44:16.098041053 +0200
+++ rustc-1.72.0-src/compiler/rustc_target/src/spec/targets/mips64_unknown_linux_gnuabi64.rs	2023-08-29 19:46:58.846059036 +0200
@@ -11,7 +11,6 @@
             abi: "abi64".into(),
             endian: Endian::Big,
             // NOTE(mips64r2) matches C toolchain
-            cpu: "mips64r2".into(),
-            features: "+mips64r2,+xgot".into(),
+            cpu: "mips4".into(),
             max_atomic_width: Some(64),
             mcount: "_mcount".into(),

